petastorm/spark/spark_dataset_converter.py

```diff
@@ -483,6 +480,47 @@ def _materialize_df(df, parent_cache_dir_url, parquet_row_group_size_bytes,
     return save_to_dir_url
 
 
+def _wait_for_fs_eventually_consistency(url_list):
+    """
+    This is used in databricks runtime. Waiting about 30 seconds to make sure
+    all files can be accessed by dbfs fuse.
+    This is because of s3 eventually consistency, and the backend of dbfs is s3.
+    """
+    fs, path_list = get_filesystem_and_path_or_paths(url_list)
+
+    wait_seconds = _get_spark_session().conf \
```

Comment:
Let's add some `logger.debug` statement to simplify debugging if a user is wondering why their app is 'hanging' for awhile?